api.py

```diff
@@ -727,6 +728,11 @@ def clear_dataset(**kwargs):
         else:
             return_message = "No matching datasets found."
 
+    for response in responses:
+        if 'location' in response.headers:
+            job_id = response.headers['location']
+            response = wait_for_job(job_id, **kwargs)
```

Comment:
just noting for posterity that this will take at a minimum `responses.count * 0.5` seconds to complete, because the first thing wait_for_job does is sleep for half a second before checking to see if the job is finished.

Reply:
Gross.  Thanks for pointing that out.  Could probably use a good exponential backoff strategy there.